src/dlgprefreplaygaindlg.ui

```diff
@@ -155,8 +158,11 @@
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_boost">
+        <property name="toolTip">
+         <string>After ReplayGain is applied, adjust the inital boost for a target loudness in LUFS (Loudness Units relative to Full Scale).</string>
```

Comment:
The scale is now absolute. 
"For tracks with ReplayGain, adjust the target loudness to this LUFS value (Loudness Units relative to Full Scale)."